x-pack/plugins/code/server/lsp/proxy.ts

```diff
@@ -105,7 +111,9 @@ export class LanguageServerProxy implements ILanguageServerHandler {
     return await clientConn
       .sendRequest(
         'initialize',
-        initOptions ? { ...params, initializationOptions: initOptions } : params
+        initOptions && initOptions.initialOptions
+          ? { ...params, initializationOptions: initOptions.initialOptions }
```

Comment:
I think 

initializationOptions
and 
clientCapabilities

are both in the root level of the request param?